bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java

```diff
@@ -226,11 +228,22 @@ public void controlResized(ControlEvent e) {
 		}
 	};
 
-	private FocusListener overlayFocusListener = FocusListener.focusLostAdapter(e -> {
-			findReplaceLogic.activate(SearchOptions.GLOBAL);
-			searchInSelectionButton.setSelection(false);
-	});
+	/**
+	 * Updates the find replace dialog on activation changes.
```

Comment:
This listener is about the "overlay", not the "dialog". I think the comment is even obsolete, as an activation listener obviously listens to activation changes, so I think you can remove the comment.